### PR TITLE
Fixes #134 : Added FOSSAIA's Company LinkedIn link, Added missing social links 

### LINKED ISSUE #134: Add LinkedIn Profiles for FOSSASIA, Add missing social links
> In the `index.html`, the codebase contains a line : `<!-- <a href="#"><i class="icon social_linkedin"></i></a> -->`
* Add the LinkedIn company link.

* Add missing social links

### PR DESCRIPTION
**Pull request by 0x48piraj**
# Fixes #134

* **_Added @fossasia company link, as it was commented in the codebase :wink:_** 
- _In the head_
- _In the body_

Added Details : 

- **arynchoong**
- **robertbobreyes**
- **libedinskylab**

**_Preview link - https://0x48piraj.github.io/2018.fossasia.org_**

Ping @mariobehling :+1:

### CODE REVIEW

**Comment by 0x48piraj** [comment]
@mariobehling @hpdang @fossasia/web-developers 

This PR is staling. Please merge!

File: index.html [hunk 1]
```diff
@@ -91,6 +91,7 @@
                                         </ul>
                                     </li>
                                     <li class="social-link"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
+                                    <li class="social-link"><a target="_self" href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a></li>
                                     <li class="social-link"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
                                     <li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
 				    <li class="social-link"><a target="_self" href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a></li>
```
Comment (SaptakS) [inline index.html]: I think the above link that you have added is the same as this line.

File: index.html [hunk 2]
```diff
@@ -1535,7 +1536,7 @@ <h1>Event Partners</h1>
 											<a href="https://github.com/fossasia"><i class="icon fa fa-github"></i></a>
 											<a href="https://www.facebook.com/fossasia/"><i class="icon social_facebook"></i></a>
 											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
-											<!-- <a href="#"><i class="icon social_linkedin"></i></a> -->
+											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
```
Comment (SaptakS) [inline index.html]: This line seems to be same as [L-1538](https://github.com/fossasia/2018.fossasia.org/pull/139/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR1538)

Comment (0x48piraj) [reply index.html]: There were two instances.

Comment (SaptakS) [reply index.html]: PFA the screenshot. It's repeated. The screenshot was taken from the test link provided by you above.

screen shot 2018-01-06 at 7 36 44 pm

Comment (0x48piraj) [reply index.html]: :open_mouth: 

Thanks!

**Comment by 0x48piraj** [comment]
@SaptakS Thanks !! :blush: 
I think, It's now all right.

File: index.html [hunk 3]
```diff
@@ -1535,7 +1535,7 @@ <h1>Event Partners</h1>
...
```
Comment (ParthS007) [inline index.html]: @0x48piraj  Why there are two similar accounts?

Comment (0x48piraj) [reply index.html]: One is for the top and one is for below.

**Comment by 0x48piraj** [comment]
@mariobehling @edenyay

**Pull request merged**